app/Http/Utils/Filters/Filter.php

```diff
@@ -634,4 +634,22 @@ public function getOriginalExp()
     {
         return $this->originalExp;
     }
+
+    /**
+     * @param FilterElement[] $e
+     * @return array
+     */
+    public static function getValueFromFilterElement(array $e): array {
```

Comment:
i am no so sure that this should be a static method
IMHO should be an instance method using https://github.com/OpenStackweb/summit-api/blob/fcac0ee396cc8dbf287deef825f103ed04d29107/app/Http/Utils/Filters/Filter.php#L111

Reply:
done